src/obs-plugin/plugin-main.cpp

```diff
@@ -102,8 +130,24 @@ bool obs_module_load(void)
             action->setText(obs_module_text("Start Virtual Camera"));
             obs_output_stop(output);
         } else {
-            action->setText(obs_module_text("Stop Virtual Camera"));
-            obs_output_start(output);
+            if(isSupportedResolution()){
+                action->setText(obs_module_text("Stop Virtual Camera"));
+                obs_output_start(output);
+            } else {
+                
+                stringstream msg;
+                msg << "Your output resolution not supported. Please use one of the following:" << endl;
```

Comment:
I think you need to use the localized methods for any kind of messaging here. I think that's `obs_module_text`

Reply:
oh ok.

Reply:
Actually, do you mean "I have to use" or do you mean "i should use it because it's nicer"? Because atm it works, but I could change it

Reply:
Done. Also I added the framework for localization, I didn't get it to work tho.